jetbrains-core/src/software/aws/toolkits/jetbrains/ui/ResourceSelector.kt

```diff
@@ -205,7 +198,35 @@ class ResourceSelector<T> @JvmOverloads constructor(
         LOADED
     }
 
-    private companion object {
-        val LOG = getLogger<ResourceSelector<*>>()
+    companion object {
+        private val LOG = getLogger<ResourceSelector<*>>()
+        @JvmStatic
+        fun builder(project: Project) = object : ResourceBuilder {
+            override fun <T> resource(resource: () -> Resource<out Collection<T>>): OtherOptions<T> = OtherOptions(project, resource)
+        }
+    }
+
+    interface ResourceBuilder {
```

Comment:
make this private

Reply:
can't - it's exposed in the builder `ResourceSelector.builder(project)`

Reply:
Right, got lost with the line wrapping on github